lesson_9/HW9/index9HW.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="shortcut icon" type="image/x-icon" href="logo.png" >
+    <meta charset="UTF-8" />
+    <title>CV_Bielenko_Front-End Developer HW7</title>
+    <link href="styles.css" rel="stylesheet" type="text/css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Alice&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="flex-cont">
+      <div class="h-text">
+        <h1 class="name">Alisa Bielenko</h1>
+        <h2 class="h2-text">Junior Front-End Developer</h2>
+      </div>
+      <div class="photo">
+        <img class="my_photo" src="photo.jpg" alt="My photo">
+      </div>
+    </div>
+    <h2 class="h2-text"><u>Work Experience:</u></h2>
+      <table class="table">
+        <tr>
+          <th class="title">Period</th>
+          <th class="title">Position</th>
+        </tr>
+        <tr>
+          <td>2005-2018</td>
+          <td class="position">Chief Accountant at <a href="https://www.google.com/">IRSM LTD</a></td>
+        </tr>
+        <tr>
+          <td>2018-present</td>
+          <td class="position">Chief Accountant at <a href="https://www.google.com/">TEXNO-CENTER LTD</a></td>
+        </tr>
+      </table>
+
+      <div class="s">
```

Comment:
rename to skills-container or something like that
classes should have some clear name to understand other people what this class is used for

Reply:
ok